class-events-manager-securesubmit.php

```diff
@@ -578,7 +579,19 @@ function authorize_and_capture($EM_Booking){
         //Return transaction_id or false
         return apply_filters('em_gateway_securesubmit_authorize', $result, $EM_Booking, $this);
     }
-
+    /**
+     * set secret key and load service configuration
+     * @return type*
+     */
+    public function setConfig()
+    {
+        $config = new ServicesConfig();
+        $config->secretApiKey = get_option('em_'.$this->gateway.'_secret_key');
+        $config->serviceUrl = "https://cert.api2.heartlandportico.com";
```

Comment:
The URL needs to support production as well, so either we need to manually switch it to `https://api2.heartlandportico.com` or defer to the SDK in recent versions of the SDK.

Reply:
updated